docs/user-guide/modify-timing-data.ipynb

```diff
@@ -42,36 +44,35 @@
-    "|3|R1|gasboiler|4|0|0|1.5|0|0|\n",
-    "|4|R1|gasboiler|5|0|0|3|0|0|\n",
-    "|5|R1|gasboiler|6|0|0|2|0|0|\n",
-    "|**6**|**R1**|**gasboiler**|**7**|**0**|**0**|**2**|**0**|**0**|\n",
-    "|**7**|**R1**|**gasboiler**|**8**|**0**|**0**|**2**|**0**|**0**|\n",
-    "|0|R2|gasboiler|1|0|0|1|0|0|\n",
-    "|1|R2|gasboiler|2|0|0|1.5|0|0|\n",
-    "|2|R2|gasboiler|3|0|0|1|0|0|\n",
-    "|3|R2|gasboiler|4|0|0|1.5|0|0|\n",
-    "|4|R2|gasboiler|5|0|0|3|0|0|\n",
-    "|5|R2|gasboiler|6|0|0|2|0|0|\n",
-    "|**6**|**R2**|**gasboiler**|**7**|**0**|**0**|**2**|**0**|**0**|\n",
-    "|**7**|**R2**|**gasboiler**|**8**|**0**|**0**|**2**|**0**|**0**|\n",
+    "|R1|gasboiler|1|0|0|**0.7**|0|0|0|\n",
+    "|R1|gasboiler|2|0|0|**1.0**|0|0|0|\n",
+    "|R1|gasboiler|3|0|0|**0.7**|0|0|0|\n",
+    "|R1|gasboiler|4|0|0|**1.0**|0|0|0|\n",
+    "|R1|gasboiler|5|0|0|**2.1**|0|0|0|\n",
+    "|R1|gasboiler|6|0|0|**1.4**|0|0|0|\n",
+    "|**R1**|**gasboiler**|**7**|**0**|**0**|**1.4**|**0**|**0**|**0**|\n",
+    "|**R1**|**gasboiler**|**8**|**0**|**0**|**1.4**|**0**|**0**|**0**|\n",
+    "|R2|gasboiler|1|0|0|**0.7**|0|0|0|\n",
+    "|R2|gasboiler|2|0|0|**1.0**|0|0|0|\n",
+    "|R2|gasboiler|3|0|0|**0.7**|0|0|0|\n",
+    "|R2|gasboiler|4|0|0|**1.0**|0|0|0|\n",
+    "|R2|gasboiler|5|0|0|**2.1**|0|0|0|\n",
+    "|R2|gasboiler|6|0|0|**1.4**|0|0|0|\n",
+    "|**R2**|**gasboiler**|**7**|**0**|**0**|**1.4**|**0**|**0**|**0**|\n",
+    "|**R2**|**gasboiler**|**8**|**0**|**0**|**1.4**|**0**|**0**|**0**|\n",
     "\n",
     "The `ProcessName` must be reported, but it is not binding on the results. It is just the way that the model reads the input data.\n",
     "\n",
-    "We do the same for the ```Residential2050Consumption.csv```, however this time we make the demand for heat in 2050 to both be 5 for the new timeslices. See [here](https://github.com/SGIModel/MUSE_OS/blob/main/docs/tutorial-code/4-modify-timing-data/1-modify-timeslices/technodata/preset/Residential2050Consumption.csv) for the full file.\n",
+    "We do the same for the ```Residential2050Consumption.csv```, but set the demand in 2050 to be triple that of 2020 in every timeslice. See [here](https://github.com/SGIModel/MUSE_OS/blob/main/docs/tutorial-code/4-modify-timing-data/1-modify-timeslices/technodata/preset/Residential2050Consumption.csv) for the full file.\n",
     " \n",
     "<!-- As we have modified the timeslicing arrangements there will be a change in the underlying demand for heating. This may require more electricity to service the demand for heat. Therefore, we relax the constraints for growth in ```windturbine``` in region `R2` in the ```technodata/power/Technodata.csv``` file, as well as the constraints for ```heatpump```, again in region `R2`. For the full power sector ```Technodata.csv``` file click [here INSERT LINK HERE](github-power-technodata), and for the full residential sector ```Technodata.csv``` file click [here INSERT LINK HERE](github-residential-technodata). -->"
```

Comment:
I know you haven't modified this line, but as it's commented out, maybe we should drop it:

```suggestion
```